<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QMainWindow" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="locale">
   <locale language="Russian" country="Russia"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_img">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>341</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string>Выберите картинку для анализа</string>
    </property>
    <widget class="QPushButton" name="pushButton_select_img">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>30</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../data/etc/file_manager.jpg</normaloff>../data/etc/file_manager.jpg</iconset>
     </property>
    </widget>
    <widget class="QLabel" name="label_img">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>241</width>
       <height>231</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_path_img">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>281</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_model">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Выберите файл с расширением .h5</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_path_model">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>281</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_select_model">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>30</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../data/etc/file_manager.jpg</normaloff>../data/etc/file_manager.jpg</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_analysis">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>281</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string>Обработка кадра нейронной моделью</string>
    </property>
    <widget class="QPushButton" name="pushButton_analyze">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>191</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Анализировать фото</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_info_res">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>90</y>
       <width>151</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Результат обработки</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_origin_res">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>221</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Оригинал: </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_spoof_res">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>221</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Копия: </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_result">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>221</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Вывод:&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_ceate_model">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>320</y>
      <width>281</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Работа с нейронной сетью</string>
    </property>
    <widget class="QPushButton" name="pushButton_create_model">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>241</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Создать модель</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
